juju/juju
# [JUJU-1108] Refactor --watch flag for 'juju status'

Refactored the implementation of the `--watch` flag for the `juju status` command. Provided the `--watch` flag implementation via the viddy module (https://github.com/sachaos/viddy).

## QA steps

```sh
juju bootstrap localhost ldx
juju deploy tiny-bash
juju status --watch 3s
```
Make sure that you see the output of the viddy module (analog of the `watch` command)

## Documentation changes

None

## Bug reference

None
